src/component/helpers/array-utils.ts

```diff
@@ -33,3 +33,13 @@ export function ensureIsArray<T>(elements: (T | string)[] | T | string, acceptEm
   }
   return returnedArray.filter(value => value);
 }
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
❓ can we add a short comment that justify the rules disabling?
I guess we expect the array store the same types, so we may use `generic type parameters` instead of `any`

Reply:
Good catch 😉